src/atoms/Button/Button.js

```diff
@@ -18,6 +18,10 @@ type ButtonProps = {|
   loading?: boolean,
  /** then true when disable button */
   disabled?: boolean,
+ /** then button is squared */
+  squared?: boolean,
+ /** then button is rounded */
+  rounded?: boolean,
```

Comment:
Maybe it's worth to set rounded prop to the true by default?

Reply:
In most cases we are using button with `rounded=false`